unit/src/ParsedFunctionTest.C

```diff
@@ -34,10 +34,12 @@ ParsedFunctionTest::init()
 
   InputParameters mesh_params = _factory->getValidParams("GeneratedMesh");
   mesh_params.set<MooseEnum>("dim") = "3";
+  mesh_params.set<std::string>("name") = "mesh";
```

Comment:
Hmm, Should we auto-generate a default name instead of making the user set one? If we make or allow the user to set one how can we guarantee that it will be unique?